README.md

```diff
@@ -2,3 +2,12 @@
 
 # pyret-ide
 UI Components that power the pyret ide
+
+React Storybook Use: To use React Storybook run the command 'npm run storybook'
```

Comment:
It's a markdown file! Use markdown syntax so that it looks nice: https://daringfireball.net/projects/markdown/